### PR TITLE
Move Encrypting Your Data

### PR DESCRIPTION
**Pull request by maria-robobug**
https://issues.couchbase.com/browse/DOC-9578

### CODE REVIEW

**Comment by thejcfactor** [comment]
LGTM!

**Review by thejcfactor — APPROVED** [review]

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -18,6 +18,7 @@
 .Advanced Data Operations
 * xref:howtos:concurrent-async-apis.adoc[Async APIs]
 * xref:howtos:concurrent-document-mutations.adoc[Concurrent Document Mutations]
+* xref:howtos:encrypting-using-sdk.adoc[Encrypting Your Data]
 * xref:howtos:transcoders-nonjson.adoc[Transcoders & Non-JSON]
 //* xref:howtos:durability.adoc[Durability]
```
Comment (osfameron) [inline modules/ROOT/nav.adoc:23]: realise this is commented out but just noticed it's out of order :D

**Review by osfameron — APPROVED** [review]
👍

File: modules/ROOT/nav.adoc (hunk 1 above)
Comment (maria-robobug) [reply modules/ROOT/nav.adoc:23]: Probably an old page that isn't used anymore? Not sure, but leaving out of this change.

**Pull request merged**